etc/packaging/appimages/cartographer-module-x86_64.yml

```diff
@@ -7,13 +7,10 @@ script:
  - mkdir -p $TARGET_APPDIR/usr/share/cartographer/lua_files
  - cp ../../../bin/cartographer-module $TARGET_APPDIR/usr/bin/
  - cp ../../../viam-cartographer/build/carto_grpc_server $TARGET_APPDIR/usr/bin/
- - cp ../../../viam-cartographer/cartographer/build/libcartographer.a $TARGET_APPDIR/usr/lib/
- - cp ../../../viam-cartographer/build/libviam-cartographer.a $TARGET_APPDIR/usr/lib/
  - cp ../../../viam-cartographer/lua_files/* $TARGET_APPDIR/usr/share/cartographer/lua_files
  - mkdir -p $TARGET_APPDIR/usr/share/icons/viam/256x256/apps/
  - cp ./viam-server.png $TARGET_APPDIR/usr/share/icons/viam/256x256/apps/viam-server.png
- - go install github.com/Otterverse/aix@latest
- - cp `go env GOPATH`/bin/aix $TARGET_APPDIR/usr/bin/
+ - GOBIN=$TARGET_APPDIR/usr/bin/ go install github.com/Otterverse/aix@latest
```

Comment:
[q] why isnt this change in rdk?

Reply:
Because I wrote the RDK stuff a year and a half ago before I knew about GOBIN. :-)

(Also, it still works there... but this had a conflict with the "old" way, as GOBIN was being set to TOOL_BIN in the makefile, go the default location was no good anymore.)